### DISCUSSION TITLE
googletest cmake file

### DISCUSSION THREAD

**Discussion by tfarina** [catboost/catboost#2505] [General]
Hi,

Are there plans to add a CMakeLists.txt file to build googletest at contrib/restricted/googletest?

One like already available at https://github.com/ydb-platform/ydb/tree/main/contrib/restricted/googletest.

Thanks,

**Comment by andrey-khropov** [comment]
Why are you asking? 

If CMakeFiles are not created there it means that code in this directory need not be built either directly as a dependency of CatBoost libraries or CLI binary or tools used during CatBoost build or testing.
